docs/terminology.md

```diff
@@ -18,7 +18,7 @@ Constraints are rules that an AdonisFX solver applies during simulation to ensur
 
 **Hard Constraint**. A hard constraint defines the location of a geometry point in the tangent space of a polygon. This constraint type is similar to the attachment constraint but in this case the transformation used is the one given by the tangent space at the closest point on an external geometry.
 
-**Shape Preservation Constraint**. A shape preservation constraint defines the state of shape formed by a vertex with its adjacents on initialisation. During simulation, a shape preservation constraint will try to mantain this shape with its neighbouring vertices.
+**Shape Preservation Constraint**. A shape preservation constraint defines the state of the shape formed by a vertex with its adjacents on initialisation. During simulation, a shape preservation constraint will try to maintain the rest shape of the geometry with its neighbouring vertices.
```

Comment:
Let's make sure that this description is also ported to other sections of the docs

Reply:
Done!